ui/src/app/sidebar/sidebar.scss

```diff
@@ -1,74 +1,99 @@
 @import 'node_modules/argo-ui/v2/styles/colors.scss';
 @import '../shared/config.scss';
 
+$deselected-text: #818d94;
+
 .sidebar {
     min-width: $sidebar-width;
     width: $sidebar-width;
-    padding: 15px;
+    padding: 0 20px;
     top: 0;
     left: 0;
     position: fixed;
     height: 100vh;
     max-height: 100vh;
     color: white;
-    background-color: $silver-lining;
+    background-color: #0f2733;
     overflow: auto;
     z-index: 0;
 
+    &__container {
+        padding: 0 5px;
+    }
+
+    &__logo-container {
+        margin-top: 10px;
+        font-size: 12px;
+        color: $deselected-text;
+        margin-left: 12px;
+    }
+
     &__logo {
-        justify-content: center;
+        width: 100%;
         display: flex;
+        flex-direction: row-reverse;
         align-items: center;
         font-size: 24px;
         font-weight: 500;
-        margin-bottom: 0.5em;
+        margin: 1em 0;
+        margin-left: 7px;
 
-        img {
-            width: 40px;
-            margin-right: 7px;
+        &__character {
+            width: 42px;
+        }
+
+        &__text-logo {
+            height: 2em;
+            filter: invert(100%);
         }
     }
 
     &__version {
         cursor: pointer;
         width: 100%;
         text-align: center;
-        margin-bottom: 1em;
     }
 
     &__nav-item {
         cursor: pointer;
         display: flex;
         align-items: center;
-        font-size: 17px;
+        font-size: 16px;
+        font-weight: 500;
         $height: 2.5em;
         height: $height;
         line-height: $height;
-        color: $dull-shine;
+        color: $deselected-text;
 
         i {
             font-size: 26px;
             color: white;
-            width: 35px;
+            width: 40px;
+            padding-right: 45px;
         }
 
         i.fa {
             font-size: 22px;
-            margin-left: 3px;
+            margin-left: 2px;
+            margin-right: -2px;
             margin-top: 12px;
         }
 
         &--active {
             color: white;
-            font-weight: 500;
-            box-shadow: -3px 0 0px 0 inset $sky;
+            i {
+                color: #fe733f;
+            }
         }
     }
 
     &__collapse-button {
         text-align: center;
         cursor: pointer;
-        margin-top: 1em;
+        margin-left: 15px;
```

Comment:
```suggestion
        margin-left: -8px;
```

Reply:
Hi Remington, try the change at line 34 and here at line 93 together.  This way, it behaves more like 1) a hamburger button and 2) you can rapidly expand and collapse the sidebar without having to move the cursor.

Reply:
I tried this, and while it has the affect of not needing to move the mouse, when in the expanded state it doesn't adhere to @ropsii's design and personally I don't think it looks as good. @ropsii can correct me if I'm wrong, but I think the idea with the arrow being on the right was for the Argo icon to align with the Nav icons.

Since it's a fairly rare case to expand and contract the sidebar multiple times in rapid succession, I think we should prioritize the design rather than keeping the arrow in place.

Reply:
Ok, that's fine if it should adhere to the original design suggestion.